test/test.dart

```diff
@@ -374,6 +374,18 @@ Future<User> loginWithRetry(App app, Credentials credentials, {int retryCount =
   }
 }
 
+Future<void> waitForCondition(bool Function() condition,
+    {Duration timeout = const Duration(seconds: 10), Duration retryDelay = const Duration(milliseconds: 100), String? message}) async {
```

Comment:
I would rather choose a small default and if more is needed (the rare case) than its required to specify one.

Reply:
Just so the frequent usages with 1 sec duration do not require to specify the timeout everytime